villains.sql

```diff
@@ -0,0 +1,170 @@
+CREATE DATABASE disney;
+
+USE disney;
+
+CREATE TABLE villains
+(
+  name VARCHAR(255),
+  movie VARCHAR(255),
+  slug VARCHAR(255),
+  createdAt DATETIME DEFAULT CURRENT_TIMESTAMP,
+  updatedAt DATETIME DEFAULT CURRENT_TIMESTAMP
+  ON
+  UPDATE CURRENT_TIMESTAMP,
+    deletedAt DATETIME,
+    PRIMARY KEY
+  (name)
```

Comment:
An `id` field would be a better primary key. Primary keys need to be unique and name typically doesn't guarantee uniqueness. `slug` would be a better choice if you wanted to natural key.